### DISCUSSION TITLE
Importing files from the aissquare.com datasets

### DISCUSSION THREAD

**Discussion by bananenpampe** [deepmodeling/dpdata#440] [Q&A]
Hello together,

I am trying to import one of the datasets from the aissquare datasets using:
```python

datadir = "./H2O-Phase-Diagram/2020.09.final.clean_v2/2020.09.final.clean/data/data.init/"
data = dpdata.MultiSystems().load_systems_from_file(datadir, fmt="deepmd/npy")
```
The import works, but for some reason it does not recognize the typemap and assignes own atom types:

```python
data.atom_names 
# -> ['Type_0', 'Type_1']

```

This than leads to issues, when for example writing the MultiSystems to disk in the ase.atoms format, because ASE cannot convert "Type_0".

Is there something I am doing wrong? 
Thank you in advance!

**Comment by bananenpampe** [comment]
okay, so I was able to resolve this by passing my own custom typemap
```python
dpdata.MultiSystems().load_systems_from_file(datadir, type_map=["O","H"],fmt="deepmd/npy")
```

I am still surprised, that it does not recognize the type_map files eventhough they are present in the directories.

**Reply by wanghan-iapcm** [reply]
I find no `type_map.raw` in the folders `2020.09.final.clean_v2/2020.09.final.clean/data/data.init/*`

**Comment by njzjz** [comment]
For AISSquare issues, you can report or contribute to https://github.com/deepmodeling/AIS-Square if the documentation is incorrect.
